publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dto/Validatable.java

```diff
@@ -0,0 +1,23 @@
+package no.unit.nva.publication.doi.dto;
+
+import static java.util.Objects.isNull;
+
+public abstract class Validatable {
```

Comment:
Probably want to use composeability for this instead of inheritance, but good enough for now.